src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -108,11 +108,17 @@ object ExtractionPipeline {
       linkingEntities = replicates
     )
 
+    val fcExperiments = extractLinkedEntities(
+      entityToExtract = EncodeEntity.FunctionalCharacterizationExperiment,
+      matchingField = "experiment",
+      linkingEntities = replicates
+    )
+
     // don't need to use files apart from storing them, so we don't assign an output here
     extractLinkedEntities(
       entityToExtract = EncodeEntity.File,
       matchingField = "@id",
-      linkingEntities = experiments,
+      linkingEntities = SCollection.unionAll([experiments, fcExperiments]),
```

Comment:
Looks like you've got Python on the brain!

In Scala you build collections via `apply` on their companion objects. For example, this could be:
```suggestion
      linkingEntities = SCollection.unionAll(List(experiments, fcExperiments)),
```

Reply:
Thanks! Just figured that out with Raaid the hard way lol.